client/src/pages/Rental/Rent.js

```diff
@@ -0,0 +1,274 @@
+                <MenuItem value={30}>Sports</MenuItem>
+              </Select>
+            </FormControl>
+          </Grid>
+          <Grid item className={classes.filter0}>
+            <FormControl variant="outlined" className={classes.formControl}>
+              <InputLabel id="driverFilter">Driver</InputLabel>
+              <Select
+                labelId="demo-simple-select-outlined-label"
+                id="demo-simple-select-outlined"
+                label="Age"
+                className={classes.select}
+              >
+                <MenuItem value="">
+                  <em>None</em>
+                </MenuItem>
+                <MenuItem value={10}>Driver 25+</MenuItem>
+                <MenuItem value={20}>Driver 30+</MenuItem>
+                <MenuItem value={30}>Driver 50+</MenuItem>
+              </Select>
+            </FormControl>
+          </Grid>
+          <Grid
+            item
+            className={classNames(classes.filter0, classes.filter0Last)}
+          >
+            <FormControl variant="outlined" className={classes.formControl}>
+              <InputLabel id="demo-simple-select-outlined-label">
+                Price
+              </InputLabel>
+              <Select
+                labelId="price"
+                id="demo-simple-select-outlined"
+                label="Age"
+                className={classes.select}
+              >
+                <MenuItem value="">
+                  <em>None</em>
+                </MenuItem>
+                <MenuItem value={10}>Lower Price First</MenuItem>
+              </Select>
+            </FormControl>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid
+        container
+        spacing={3}
+        justify="center"
+        className={classes.gridContainer}
+      >
+        {CarList.map((car) => (
+          <Grid item key={car.name} xs={12} sm={3}>
+            <Paper elevation={3} className={classes.paperContainer}>
+              <div className={classes.headerInformation}>
+                <div className={classes.bottomInformationContainer}>
+                  <span className={classes.legend1}>{car.name}</span>
+                  <span
+                    className={classNames(
+                      classes.subLegend1,
+                      classes.comment
+                    )}
+                  >
+                    {car.comment}
+                  </span>
+                </div>
+              </div>
+              <div className={classes.imageContainer}>
+                <img className={classes.image} src={car.url} alt="carImage" />
+              </div>
+              <div className={classes.bottomInformationContainer}>
+                <span className={classes.legend1}>
+                  ${car.price}
+                  <span className={classes.subLegend1}>/per day</span>
+                </span>
+                <span className={classes.legend2}>${car.total} Total</span>
+              </div>
```

Comment:
Same as above: try to use more Material UI components for this instead of plain JSX.